modules/monitoring-installation-progress.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting-installations.adoc
+
+[id="monitoring-installation-progress_{context}"]
+= Monitoring installation progress
+
+Monitor high level installation, bootstrap and control plane logs as an {product-title} {product-version} installation progresses. This provides greater visibility into how an installation progresses and helps identify the stage at which an installation failure occurs.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role. Note that the initial kubeadmin password can be found in `<install_directory>/auth/kubeadmin-password` on the installation host.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* The `ssh-agent` process is active on your computer, and you provided both the `ssh-agent` process and the installation program the same SSH key.
+* If you tried to install a cluster on infrastructure that you provisioned, you must have the fully-qualified domain names of the control plane, or master, machines.
+
+.Procedure
+
+. Watch the installation log as the installation progresses:
++
+----
+$ tail -f ~/<installation_directory>/.openshift_install.log
+----
+
+. Monitor the `bootkube.service` journald unit log on the bootstrap node, once it has booted (after running `./openshift-install --dir=install_dir wait-for bootstrap-complete`). This provides visibility into the bootstrapping of the first control plane. Replace `<cluster_name>.<sub_domain>.<domain>` with appropriate values:
```

Comment:
It's unclear what you mean by the "after running ..." command in parentheses. I don't see anything in a prior step or prereq that says to run this command.

Reply:
`./openshift-install --dir=install_dir wait-for bootstrap-complete` is the initial command run to start a bare metal OpenShift installation. On reflection, that part of the sentence is superfluous and I have now removed it.